packages/lib-utils/src/components/status/StatusBox.tsx

```diff
@@ -0,0 +1,72 @@
+// @ts-nocheck
```

Comment:
What issues did we run into with typescript here? I don't think we could merge this without TS checking.

Reply:
This code is so far just a draft. I'll remove this. It's just for dev purposes to prevent it from causing build errors.